Exercise_1.py

```diff
@@ -1,18 +1,29 @@
+Time complexity: O(1) for isEmpty, push, pop, peek. O(n) for size and show.
+Space complexity: O(1) for all
+Executed in Leetcode: No link. Code executed in jupyter notebook.
+Challenges: Not sure about time and space complexity.  Can we use size() function in peek?
```

Comment:
To get the length of a list, in python there's len() function.

Reply:
I have used the len() function inside the size function.